newcomers-tutorial.md

```diff
@@ -54,20 +54,17 @@ opt to skip this part for the time being.
   `texlive-fonts-recommended`, `fonts-lmodern`, `texlive-latex-recommended`,
   `texlive-latex-extra` - refer to your distro for available packages)
 
-### Required Bureaucracy
+### Required Accounts
 
-GHC uses its own gitlab environment, found at
-[gitlab.haskell.org](https://gitlab.haskell.org/). In order to submit merge
-requests, file issues, and participate in code reviews, you will need to create
-an account, or sign in with a GitHub account.
+In order to submit merge requests, file issues, and participate in code reviews, you will need to create a gitlab account, or sign in with GitHub account. GHC uses its own gitlab environment, found at [gitlab.haskell.org](https://gitlab.haskell.org/).
 
 ## Getting The Code
 
-Since we will be committing patches, we will need a GitLab fork, so it's best
-to get that out of the way right away. So sign into GitLab, find [the GHC
+Since you will be committing patches, you will need a GitLab fork, so it's best
+to get that out of the way right away. Sign into GitLab, find [the GHC
 project](https://gitlab.haskell.org/ghc/ghc), and fork it.
 
-Then clone it to your development machine:
+Then, clone it to your development machine:
```

Comment:
I was trying to improve the flow. Starting a new paragraph with `Then` is kind of stilted. I was also trying not to rework your writing too much! :) (I prefer not to rework someone's writing too much when I'm editing, because it's their writing I'm editing).

I'll have a think and see if I can come up with something that meets us both half way :)

Reply:
Update: okay, I significantly reduced the pargraph. Hopefully I haven't offended you too much with my edits.